### ISSUE TITLE
Upload took too long and got aborted

### ISSUE THREAD

**Issue by sachaqs** [keybase/client#4178]
"The write operation took too long and failed"

my log id: 50102d02fb211e7d2e10f11c

**Comment by strib** [comment]
Hmm, sorry about that.

**Comment by strib** [comment]
(Oops, finger slipped and hit enter too soon.)

Looks like your network throughput to our servers was a bit slow, and then your kernel did a 2MB write all at once, which your network couldn't handle within the OSX timeout period.  That's unfortunate.

We are working on rearchitecting a bit so that writes like this aren't dependent on your network speed -- it will be ready in a couple months.  Until then, I'm not sure there's much we can do besides suggest you try again with a better connection.  Sorry!

**Comment by sachaqs** [comment]
Oopsie. I totally forgot to reply. Sorry. :-/
Anyway.
Though I indeed don't have a big upstream bandwidth, it's not that slow: 1 Mbps. During this transfer the throughput wasn't even maxed out (@ about 0.5 Mbps). I don't know if it was on my side or yours but the error popped-up during the transfert and the file did eventually get through. 🎱 

I just made the same test with the same file (around 5 MB) this time @ 1 Mbps. And I still got the same error. I guess will have to be a little more patient and wait for the rearchitecting you mentioned but I thought that saying that the threshold taht triggers the error may be a lil'bit low could be useful. :)
